internal/docs/exported_fields.go

```diff
@@ -25,7 +25,8 @@ type fieldsTableRecord struct {
 var escaper = strings.NewReplacer("*", "\\*", "{", "\\{", "}", "\\}", "<", "\\<", ">", "\\>")
 
 func renderExportedFields(fieldsParentDir string) (string, error) {
-	validator, err := fields.CreateValidatorForDirectory(fieldsParentDir)
+	validator, err := fields.CreateValidatorForDirectory(fieldsParentDir,
+		fields.WithDisabledImportAllECSSChema())
```

Comment:
We use `CreateValidatorForDirectory` in more places, shouldn't the import be disabled there too? Unless mappings are included in the package.

Could you add a test package that misses an ECS field definition, doesn't import it and fails validation?

Reply:
> Could you add a test package that misses an ECS field definition, doesn't import it and fails validation?

The test packages that we have currently are expected to be run/validated successfully. Because of that I've added a test `TestValidate_WithDisabledImportAllECSSchema` that checks that. Would that be ok ?

Reply:
> The test packages that we have currently are expected to be run/validated successfully.

Ah, you are right. We should have a way to add negative tests too, let's leave this for a future change. 

> Because of that I've added a test `TestValidate_WithDisabledImportAllECSSchema` that checks that. Would that be ok ?

Yes, this will make it by now.